draft-ietf-quic-tls.md

```diff
@@ -321,8 +321,9 @@ data into the same packet.
 In general, the rules for which data can appear in packets of which
 encryption level are the same in QUIC as in TLS over TCP:
 
-u- Handshake messages MAY appear in packets of any encryption level.
-  [TODO: Alerts]
+- CRYPTO frames (Handshake messages) MAY appear in packets of any encryption level.
```

Comment:
nit: I prefer without parens "CRYPTO frames containing handshake messages MAY appear..."

Is it valid to send CRYPTO frames in 0RTT?

Reply:
The problem is that this isn't restrictive. CRYPTO frames only contain HS messages.